index.html

```diff
@@ -123,50 +126,76 @@ <h1 class="w-100 dark-gray f2 tc">
 </li><li class="f3 lh-copy pv2 mid-gray">Text-mining ready corpus.</li>
 </ul>
 </section>
-  <section id="technology" class="w-100 center bg-dark-gray pa3 mv3">
-  <h1 class="w-100 light-gray f2 tc">
+    
+<section id="starter" class="w-100 center bg-dark-gray pa3 mv3 tl">
+  <h1 class="light-gray f2 tc">Getting Started</h1>
+  <p class="f3 w-80-ns lh-copy ph5-ns light-gray center">When you download and install ScienceFair, you'll be able to search the eLife journal (more sources to come soon!), download articles locally, and read and tag them. Here's how that looks!</p>
+    
+  <div class="dt-ns dt--fixed-ns">
+  <div class="dtc-ns tc pv4">
+    <h2 class="light-gray">1. Search & Discovery</h2>
+    <p class="f3 lh-copy ph5-ns light-gray center">ScienceFair provides an instant search of your local collection and remote sources. Search on whatever terms you'd like!</p>
+    <img src="assets/discovery.gif" class="w-90-ns">
+  </div>
+  <div class="dtc-ns tc pv4">
+    <h2 class="light-gray">2. Exploring the Lit</h2>
+    <p class="f3 lh-copy ph5-ns light-gray center">ScienceFair has an in-app reader, putting key aspects of articles such as references and figures into the sidebar for convenience.</p>
+    <img src="assets/reading.gif" class="w-90-ns">
+  </div>
+  <div class="dtc-ns tc pv4">
+    <h2 class="light-gray">3. Tagging</h2>
+    <p class="f3 lh-copy ph5-ns light-gray center">ScienceFair gives users the freedom to bulk edit metadata, as well as tag materials with their own keywords for quicker, easier access.</p>
```

Comment:
We don't actually provide metadata editing - but we do visualise it. Maybe replace 'edit' with 'visualise'?